reader.go

```diff
@@ -1578,3 +1578,14 @@ func extractTopics(members []GroupMember) []string {
 
 	return topics
 }
+
+func describeOffset(offset int64) string {
+	switch offset {
+	case FirstOffset:
+		return "(first offset)"
+	case LastOffset:
+		return "(last offset)"
+	default:
+		return fmt.Sprintf("%d", offset)
```

Comment:
How about replacing this with `strconv.FormatInt` for stronger type safety?
```suggestion
		return strconv.FormatInt(offset, 10)
```

Reply:
Done.